### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by caycehouse**
I created a README.md for this project as I hoped it would facilitate better communication with what the repository holds to people who are not familiar with the project. 🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+![](https://runelite.net/img/logo.png)
+# launcher [![Travis](https://img.shields.io/travis/runelite/launcher.svg)](https://travis-ci.org/runelite/launcher) [![Discord](https://img.shields.io/discord/301497432909414422.svg)](https://discord.gg/mePCs8U)
+
+This repository holds the launcher for the runelite client, [click here](https://github.com/runelite/runelite) if you are looking for the runelite client.
```
Comment (Nightfirecat) [inline README.md]: `click here` is never a good link text. I'd recommend rewording this to be:

```
This repository holds the launcher for the [RuneLite client](https://github.com/runelite/runelite).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+If you have any questions, please join our IRC channel on [irc.rizon.net #runelite](http://qchat.rizon.net/?channels=runelite&uio=d4) or alternatively our [Discord](https://discord.gg/mePCs8U) server.
```
Comment (Nightfirecat) [inline README.md]: Discord might be better to mention first? I think we have more users there.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+### License
+
+RuneLite is licensed under the BSD 2-clause license. See the license header in the respective file to be sure.
```
Comment (Nightfirecat) [inline README.md]: This also could use some rewording.

```
RuneLite is [licensed under the BSD 2-clause license](https://github.com/runelite/launcher/blob/master/LICENSE).
```

I don't feel it's too important to mention the file license headers.

**Comment by caycehouse** [comment]
@Nightfirecat Sounds good, I updated the README.

**Comment by axelsly** [comment]
Unless this is a base/starter README to get the ball rolling, @caycehouse do you mind filling in more details? The README doesn't tell me much and what it does tell me is pretty self explanatory from the github repo name.

**Pull request merged**